### ISSUE TITLE
Kill the rule suggesting the use of specific reduce block argument names

### ISSUE THREAD

**Issue by vitormd** [bbatsov/rubocop#3725]
I was about to complain about the rule suggesting the use of `|a, e|` as block params name for `reduce` and so I went to the style-guide used by rubocop and the last commit is this one:
[commit](https://github.com/bbatsov/ruby-style-guide/commit/b2a8afd834737e92b4120316b407e7e6f99bf0d4).

Bottom line:
- A user should be allowed to choose other names for `reduce` block params. Such as
```ruby
total_jobs = jobs.reduce { |sum, job| sum + job.count }
```

I am happy to come here create this issue so it gets updated immediately.

**Comment by bbatsov** [comment]
The cop should not be killed (some people likely find it useful), but I guess we can disable it by default.
